db/primary_migrate/20230206172633_add_default_value_to_identity_aal.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultValueToIdentityAal < ActiveRecord::Migration[7.0]
+  def change
+    change_column_default :identities, :aal, 0
```

Comment:
We'll want to make note of this if it's the path we're going to take as it will require updating all of the existing rows, which might take a significant amount of time.

Reply:
This kinda nudges me towards thinking we should create a text column if that's the ultimate goal. The text -> integer -> text conversion means we lose some of the details like whether it was AAL2 phishing-resistant or AAL2 HSPD12, etc.

Reply:
maybe add a different column, since adding columns should be cheaper than changing the default value?

Reply:
I have no preference either way.

Reply:
**question:** this feels separate from the change of adding the fields back to the presenter - can you clarify what the motivation was? Thanks!

Reply:
Do we have a consensus at this point on integers vs string? Based on the [conversation in slack](https://gsa-tts.slack.com/archives/C01SU3NB9T3/p1676301424606279?thread_ts=1676296424.586489&cid=C01SU3NB9T3) it sounds like we are sticking with integers, and interpolating their meaning here.

Reply:
@zachmargolis @mitchellhenke thoughts?

Reply:
if we're in a rush, i guess using integers is fine, but as long as we note that there's a loss of fidelity and we should follow up to imrpove it later